src/compiler/scala/tools/nsc/Driver.scala

```diff
@@ -14,7 +14,8 @@ abstract class Driver {
   protected var settings: Settings = _
 
   protected def scalacError(msg: String): Unit = {
-    reporter.error(FakePos("scalac"), msg + "\n  scalac -help  gives more information")
+    val indent = "\u0020" * 2
```

Comment:
This seems superfluous unless you intend some tab-like behavior with indentation (which isn't here).  Just stick two spaces in in place of `${indent}` below?